### ISSUE TITLE
Update the EHEP hex editor to 1.1.0

### ISSUE THREAD

**Issue by dschadow** [jcryptool/core#99]
JCrypTool [EHEP plug-in](http://ehep.sourceforge.net) net.sourceforge.ehep is using EHEP 1.0. Update to version 1.1. Keep in mind that some classes may contain JCrypTool optimizations since the available plug-ins in our RCP are limited. 

The release notes indicate a German translation, which may make our net.sourceforge.ehep.nl_de fragment superfluous.

**Comment by grthor** [comment]
This problem gained some importance after we noticed that copy or paste from/to the editor is not possible.

Maybe this should be done before release 1.0?

There are two possibilities. This is EHEP 1.1 as suggested or [javahexeditor](https://sourceforge.net/projects/javahexeditor/).
As EHEP seems to be not updated since 2013 it would be better to use javahexeditor as it is still maintained.

**Comment by grthor** [comment]
I tested if copy and paste works in EHEP 1.1 in my eclipse ide. It did not work, so updating from ehep 1.0 to 1.1 won't get us an improvement.

Updating to javahexeditor is the solution to go now.

**Comment by grthor** [comment]
During the weekend I worked on the integration of the Javahexeditor into JCT. 
Now the Javahexeditor is displayed in JCT and some of the functions are already running. Already redo-, undo-, copy-, paste-, cut- and insert-commands are working. 

What does not work yet is saving files. Also trimming the content does not work.

There is still room for improvement in the GUI. For example there is no horizontal scrollbar and a lot of space is wasted on the left side of the plugin. 

I'm still working on that, though.

This is how the javahexeditor looks like in JCT.
![image](https://user-images.githubusercontent.com/20046726/87878049-365bd000-c9e2-11ea-97a6-19deee6839f9.png)

**Comment by grthor** [comment]
The editor only allows you to copy the ASCII characters. So it would be nice if you could also copy the HEX characters. But this is only a feature request with low priority.

**Comment by grthor** [comment]
The migration to javahexeditor is mostly finished. Therefore I will close this issue. More news about migration can be found in the PR: https://github.com/jcryptool/core/pull/222
